web-ui/src/components/template-card/TemplateCard.css

```diff
@@ -0,0 +1,53 @@
+.card{
```

Comment:
Be warned...these CSS classes will bleed. They are not magically keyed to this component like classes that are generated using the MUI templating.  You should likely prefix them all OR put a class on the top level DOM element for the class and change these to specify elements with the class `card` that are children of something that has the top level class

Reply:
thanks! it should be fixed.